index.coffee

```diff
@@ -8,15 +8,19 @@ exec = require('child_process').exec
 gutil = require 'gulp-util'
 through = require 'through2'
 
-module.exports = (options = {}) ->
+module.exports = (options = {bundle:true}) ->
 
   transform = (file, enc, callback) ->
     if file.isStream()
       error = new gutil.PluginError PLUGIN_NAME, 'Streaming not supported'
       @emit 'error', error
       return callback()
 
-    exec "bundle exec livingstyleguide compile #{file.path}", (error) =>
+    if options.bundle == false
```

Comment:
Can we switch the order? Negations sometimes are hard to read for people that are new to the codebase.

``` coffee
if options.bundle
  execStr = "bundle exec livingstyleguide compile #{file.path}"
else
  execStr = "livingstyleguide compile #{file.path}"